### ISSUE TITLE
React component only allows getting specific discussion number

### ISSUE THREAD

**Issue by YitziG** [giscus/giscus-component#6]
**Describe the bug** A clear and concise description of what the bug is.

Trying to use the React component. All attempted mappings fail with "discussion not found" even though I know it exists.

Posting a new comment creates a new discussion but even that does not show up on refresh.

**To Reproduce** Steps to reproduce the behavior:

1. Add react component
2. Use any mapping aside from number

**Expected behavior** Should work

**Desktop (please complete the following information):**

- OS: MacOS Big Sur
- Browser: Chrome
- Version: Version 92.0.4515.131 (Official Build) (x86_64)

**Comment by laymonage** [comment]
Hey, sorry for getting back so late, for some reason I wasn't subscribed to the repo.

Could you try checking the Network tab in your browser's DevTools and see the query sent to the API? What's the `term` in the query?

**Comment by Laegel** [comment]
Hey @laymonage, I think I ran into a similar problem. The React component wasn't even creating a comment, so no discussion.
In the end, I took a look at how you [implemented it on your own website](https://github.com/laymonage/base/blob/4080da98aaaca96d701972154d3038cc757bdb50/src/components/Giscus.tsx) and applied it to mine, which works.
In my case the term was `technos%2Farticles%2Fdeno` but I'm not sure that the problem comes from it, as it works with a vanilla JS version.
(thanks for Giscus BTW, using GH Discussions as a DB is smart)
